doc/Customization-guide/Code-style/Shell/index.md

```diff
@@ -4,4 +4,25 @@ title: Shell
 permalink: /:path/
 sort_idx: 50
 ---
-Specific per-language code style guidelines. Check style configuration files for Eclipse and IDEA. Style check also must be automated in the build process (after code styles are defined here).
+
+- Prefer plain Bourne shell scripts when possible. Avoid using GNU Bash, zsh or fish since they are possibly will be missing in a particular environment.
```

Comment:
Add `(sh)` after `Bourne shell`. Most people don't know, how it's called and may confuse it with bash